README.md

```diff
@@ -49,45 +49,28 @@ Now the app should be served at `http://127.0.0.1:3000`
 
 * All server functions (`#[server]`) **must** be explicitly registered (see usage sample below). In native code, Leptos uses a clever macro to register them automatically; unfortunately, that doesn't work in WASI.
 
-* When using a context value in a component in a `feature = "ssr"` block, you **must** call `use_context` **not** `expect_context`. `expect_context` will panic during routing.  E.g.
-
-```rust
-#[component]
-fn HomePage() -> impl IntoView {
-    #[cfg(feature = "ssr")]
-    {
-        if let Some(resp) = use_context::<leptos_spin::ResponseOptions>() {
-            resp.append_header("X-Utensil", "spork".as_bytes());
-        };
-    }
-
-    view! {
-        <h1>"Come over to the Leptos side - we have headers!"</h1>
-    }
-}
-```
-
 ## Usage
 
 ```rust
-use leptos::ServerFn;
-use leptos_spin::{render_best_match_to_stream, RouteTable};
-use spin_sdk::http::{ResponseOutparam, IncomingRequest};
-use spin_sdk::http_component;
-
-#[http_component]
-async fn handle_request(req: IncomingRequest, resp_out: ResponseOutparam) {
-    let mut conf = leptos::get_configuration(None).await.unwrap();
-    conf.leptos_options.output_name = "sample".to_owned();
-
-    // A line like this for every server function
-    register_explicit::<crate::app::SaveCount>();
-
-    let app_fn = crate::app::App;
-
-    let mut routes = RouteTable::build(app_fn);
-    routes.add_server_fn_prefix("/api").unwrap();
-
-    render_best_match_to_stream(req, resp_out, &routes, app_fn, &conf.leptos_options).await
+async fn handle_request(
+    request: IncomingRequest,
+    response_out: ResponseOutparam,
+) -> Result<(), HandlerError> {
+    use leptos_wasi::prelude::Handler;
+
+    let mut conf = get_configuration(None).unwrap();
+    conf.leptos_options.output_name = Arc::from("testleptos".to_owned());
```

Comment:
This whole configuration thing, much like the other one, is somewhat bypassing cargo-leptos's values for dev and the env vars people usually set for prod

Reply:
Indeed a test app seemed to work fine without this override. Pushed up the removal here + in README, good callout!